store/v2alpha1/smt/ics23.go

```diff
@@ -104,7 +104,7 @@ func convertInnerOps(path []byte, sideNodes [][]byte) []*ics23.InnerOp {
 			Hash:   ics23.HashOp_SHA256,
 			Prefix: []byte{1},
 		}
-		if getBitAtFromMSB(path[:], depth-1-i) == 1 {
+		if getBitAtFromMSB(path, depth-1-i) == 1 {
```

Comment:
Was there a reason for using `path[:]` in the first place?

Reply:
this change was made by either:

golangci-lint run ./... --fix

or 

gofumpt -w -l .

So, I am going to assume that there was not. I didn't manually change this.